src/nunavut/lang/c/templates/deserialization.j2

```diff
@@ -157,12 +157,12 @@
     offset_bits += {{ t.capacity }}UL;
 
 {# SPECIAL CASE: BYTES-LIKE ARRAY #}
-{% elif t.element_type is PrimitiveType and t.element_type.bit_length == 8 and t.element_type|is_zero_cost_primitive %}
+{% elif t.element_type is PrimitiveType and t.element_type.bit_length == 8 and t.element_type is zero_cost_primitive %}
     nunavutGetBits(&{{ reference }}[0], &buffer[0], capacity_bytes, offset_bits, {{ t.capacity }}UL * 8U);
     offset_bits += {{ t.capacity }}UL * 8U;
 
 {# SPECIAL CASE: ZERO-COST PRIMITIVES #}
-{% elif t.element_type is PrimitiveType and LITTLE_ENDIAN and t.element_type|is_zero_cost_primitive %}
+{% elif t.element_type is PrimitiveType and LITTLE_ENDIAN and t.element_type is zero_cost_primitive %}
```

Comment:
If I read you right, `LITTLE_ENDIAN` is no longer necessary here, correct? I guess it's fine to keep it for now though.